pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+python_files = djpyfs/tests.py
+addopts = --cov djpyfs --cov-report term-missing
```

Comment:
This can all go in `tox.ini` instead if you want to avoid needing the extra file (pytest looks in both places), but we've done it both ways in different repos.